app/client/index.html

```diff
@@ -61,11 +61,25 @@
     </nav>
 
     <div class="content">
-    <h3 id="featured-videos">Featured Videos</h3>
-    <div class="main-featured">
-
-    </div>
-
+        <div class="featured row">
+            <h3 id="featured-videos">Featured Videos</h3>
+            <div class="main-featured">
+                <a href="" class="video-thumb-link">
+                    <img class="main-featured-thumb video-thumb" src="temp_assets/thumbnail.png" alt="ASDF: Name of video">
+                    <p class="video-thumb-text">(3:00) Last Week Tonight with John Oliver</p>
+                </a>
+            </div>
+            <div class="side-featured col-xs-4">
+                <a href="" class="video-thumb-link">
+                    <img class="video-thumb" src="temp_assets/thumbnail.png" alt="ASDF: Name of video">
+                    <p class="video-thumb-text">(3:00) Last Week Tonight with John Oliver</p>
```

Comment:
As a stylistic preference, I think that it'd be better to list the length of the video after its title. That way, users focus on the title of the video first and not necessarily the length. Thoughts?